.travis.yml

```diff
@@ -10,10 +10,14 @@ python:
   - "2.7"
 
 # command to install dependencies
-install: "pip install -Ue .[test,build]"
+install: 
+  - "pip install -Ue .[test,build]"
+  - pip install pep8==1.5.7
 
 # command to run tests, e.g. python setup.py test
-script: nosetests --with-cov --cov=testfixtures
+script:
+  - nosetests --with-cov --cov=testfixtures
+  - pep8
```

Comment:
I'm not a massive fan of automated tools that rigidly apply a set of rules that are not designed to be applied rigidly:
http://legacy.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds